### DISCUSSION TITLE
please Please inquire about the meaning of some parameters in dataset "fox"

### DISCUSSION THREAD

**Discussion by lishuwei0424** [NVlabs/instant-ngp#373] [Q&A]
![image](https://user-images.githubusercontent.com/42164855/157999571-45b24656-0c52-4d4f-81a4-78aa56f0fb27.png)
please Please inquire about the meaning of some parameters in dataset "fox",
such as 
  "camera_angle_x": 0.7481849417937728,
  "camera_angle_y": 1.2193576119562444,
  "aabb_scale": 4,
   "sharpness": 31.752987436300323
Understand these parameters and make my datasets,thanks you very much!

**Comment by slash-under** [comment]
I'd recommend looking here to train new NeRF datasets: https://github.com/NVlabs/instant-ngp/blob/master/docs/nerf_dataset_tips.md

**Comment by lishuwei0424** [comment]
> I'd recommend looking here to train new NeRF datasets: https://github.com/NVlabs/instant-ngp/blob/master/docs/nerf_dataset_tips.md

thanks you, according to your tips,I solve the problem，but I want to  Understand these parameters ,such as  "camera_angle_x": 0.7481849417937728,
"camera_angle_y": 1.2193576119562444
,so that better to use

**Comment by neilgogogo** [comment]
"camera_angle_x": 
"camera_angle_y": 
is camera intrinsic parameters, by which you could calculate the focal length of x and y using the angle. 
```
fl = 0.5 * w / tan(0.5 * angle_x);
angle_x = math.atan(w/(fl_x*2))*2
```

**Comment by neilgogogo** [comment]
sharpness seems to be used to filter out some blurry pictures, coz there might be some burry images in a long sequence captured by a phone or camera.

**Comment by lishuwei0424** [comment]
> sharpness seems to be used to filter out some blurry pictures, coz there might be some burry images in a long sequence captured by a phone or camera.

thanks you,I get it

**Comment by lishuwei0424** [comment]
> 



> "camera_angle_x": "camera_angle_y": is camera intrinsic parameters, by which you could calculate the focal length of x and y using the angle.
> 
> ```
> fl = 0.5 * w / tan(0.5 * angle_x);
> angle_x = math.atan(w/(fl_x*2))*2
> 


 "camera_angle_x": and  "camera_angle_y" are FOV?
 I find that:
FOV_x = 2 atan(0.5 width(sensor width) / focal(mm))
FOV_y = 2 atan(0.5 height(sensor heght) / focal(mm))

**Reply by InfiniteExpression** [reply]
> "camera_angle_x": and "camera_angle_y" are FOV? I find that: FOV_x = 2 atan(0.5 width(sensor width) / focal(mm)) FOV_y = 2 atan(0.5 height(sensor heght) / focal(mm))

Is that supposed to be "2*atan" or "atan2"?
